app/controllers/api/proxy_rules_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
   private
 
   def authorize!
-    provider_can_use!(:independent_mapping_rules)
+    raise ActiveRecord::RecordNotFound unless service.independent_mapping_rules?
```

Comment:
```suggestion
    raise ActiveRecord::RecordNotFound unless service.act_as_product?
```

Reply:
Hmm. This would mean not being able to use `independent_mapping_rules` on its own (and it is already part of `master`) and force us to roll out APIAP to every account. I'd prefer having the option.

Reply:
There are no account using that on SaaS, why should we have it, and just testing it under APIAP?
Less tests and time consuming/context switching